src/main/webapp/app/pages/curation/collapsible/MutationCollapsible.tsx

```diff
@@ -0,0 +1,373 @@
+          </div>
+        </Collapsible>
+        {mutation.mutation_effect.germline && (
+          <Collapsible
+            open
+            className={'mt-2'}
+            title={'Germline'}
+            borderLeftColor={NestLevelColor[NestLevelMapping[NestLevelType.GERMLINE]]}
+            isSectionEmpty={isSectionEmpty(data, buildFirebaseGenePath(hugoSymbol, `mutations/${firebaseIndex}/mutation_effect/germline`))}
+          >
+            <RealtimeCheckedInputGroup
+              groupHeader="Pathogenic"
+              isRadio
+              options={[
+                PATHOGENICITY.PATHOGENIC,
+                PATHOGENICITY.LIKELY_PATHOGENIC,
+                PATHOGENICITY.BENIGN,
+                PATHOGENICITY.LIKELY_BENIGN,
+                PATHOGENICITY.UNKNOWN,
+              ].map(label => ({
+                label,
+                fieldKey: `mutations/${firebaseIndex}/mutation_effect/germline/pathogenic`,
+              }))}
+            />
+            <RealtimeCheckedInputGroup
+              groupHeader="Penetrance"
+              isRadio
+              options={[PENETRANCE.HIGH, PENETRANCE.INTERMEDIATE, PENETRANCE.LOW, PENETRANCE.OTHER].map(label => ({
+                label,
+                fieldKey: `mutations/${firebaseIndex}/mutation_effect/germline/penetrance`,
+              }))}
+            />
+            <RealtimeCheckedInputGroup
+              groupHeader="Mechanism of Inheritance"
+              isRadio
+              options={[GERMLINE_INHERITANCE_MECHANISM.RECESSIVE, GERMLINE_INHERITANCE_MECHANISM.DOMINANT].map(label => ({
+                label,
+                fieldKey: `mutations/${firebaseIndex}/mutation_effect/germline/inheritanceMechanism`,
+              }))}
+            />
+            <RealtimeTextAreaInput
+              fieldKey={`mutations/${firebaseIndex}/mutation_effect/germline/cancerRisk`}
+              inputClass={styles.textarea}
+              label="Cancer Risk"
+              name="cancerRisk"
+            />
+          </Collapsible>
+        )}
+      </Collapsible>
+      {mutation.tumors?.map((tumor, tumorIndex) => {
+        const cancerTypeName = tumor.cancerTypes.map(cancerType => getCancerTypeName(cancerType)).join(', ');
+        const cancerTypeFirebasePath = buildFirebaseGenePath(hugoSymbol, `mutations/${firebaseIndex}/tumors/${tumorIndex}`);
+
+        return (
+          <Collapsible
+            className={'mt-2'}
+            key={tumor.cancerTypes_uuid}
+            title={`Cancer Type: ${cancerTypeName}`}
+            borderLeftColor={NestLevelColor[NestLevelMapping[NestLevelType.CANCER_TYPE]]}
+            info={<CancerTypeLevelSummary mutationUuid={mutation.name_uuid} cancerTypeUuid={tumor.cancerTypes_uuid} />}
+            action={
+              <>
+                <span>
+                  <GeneHistoryTooltip key={'gene-history-tooltip'} historyData={parsedHistoryList} location={getMutationName(mutation)} />
```

Comment:
Could you also check if there's a non-unique key error for `gene-history-tooltip`?

Reply:
Yeah will take a look

Reply:
That wasn't, but I did notice another non-unique key error that I just fixed.